frontend/src/citizen-frontend/header/AttentionIndicator.tsx

```diff
@@ -0,0 +1,39 @@
+// SPDX-FileCopyrightText: 2017-2021 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+import React from 'react'
+import styled from 'styled-components'
+
+interface Props {
+  toggled: boolean
+  children: React.ReactNode
+  'data-qa'?: string
+}
+
+export default React.memo(function AttentionIndicator({
```

Comment:
In practice it's only used with icons but it works with other kinds of elements as well. I don't know if it's very useful but I like this component as just a wrapper since usually the icon is usually the more important element anyway.